server/utils/citations/structuredCitations.ts

```diff
@@ -2,10 +2,12 @@ import fs from 'fs';
 import path from 'path';
```

Comment:
These changes and the ones in `types/pub.ts` aren't part of the fix, I just added them while debugging since we have better types available for these values now!